code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -27,6 +27,7 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	var/volume = CELL_VOLUME //liters
 	var/last_share = 0
 	var/list/reaction_results
+	var/static/list/nonreactive_gases = list(typecacheof(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide)) // These gasses cannot react amongst themselves

```

Comment:
Indeed hardcoded inert gases are bad
WHo knows what will react witgh what in the future of chem gases

Reply:
The whole point is that this combo of gases exists all over the station the moment the round starts. So it's the vast majority of all react() tests AND its also a poor candidate for future reactions, at most you might have to throw in a temp test or something.